Readme.md

```diff
@@ -30,12 +35,12 @@ _Pseudolocale_ includes a command line interface to make it easy to incorporate
 Note: Nodejs must be installed to use the command line interface.
 
 ```bash
-./dist/cli.mjs --string 'This is going to be pseudolocalized %token%.'
+pseudolocale --string 'This is going to be pseudolocalized %token%.'
```

Comment:
you don't need to call `cli.js` directly. Thanks to `bin` field in `package.json` your package manager make it available as a command for you.